ccd-sdk-types/src/main/java/uk/gov/hmcts/ccd/sdk/types/WorkBasketResult.java

```diff
@@ -0,0 +1,26 @@
+package uk.gov.hmcts.ccd.sdk.types;
+
+import java.util.ArrayList;
+import java.util.List;
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+public class WorkBasketResult {
+  private List<WorkBasketResultField> fields;
+
+  public static class WorkBasketResultBuilder<T> {
+
+    public static <T> WorkBasketResultBuilder<T> builder() {
+      WorkBasketResultBuilder<T> result = WorkBasketResult.builder();
+      result.fields = new ArrayList<>();
+      return result;
+    }
+
+    public WorkBasketResultBuilder<T> field(String fieldName, String label) {
```

Comment:
Would we also want an overload that lets people reference fields on their model like on some other builders?

Reply:
yep, just gonna push the commit for this 👍